Jenkinsfile

```diff
@@ -87,4 +92,40 @@ pipeline {
       }
     }
   }
+  post {
+    aborted {
+      script {
+        def messageColor = 'warning'
+        if (params.BRANCH == 'latest' && GIT_BRANCH == 'latest') {
```

Comment:
Could this condition be simplified to be `if (GIT_BRANCH == 'latest')`? Same with the others below.

Reply:
Not sure - I think it's like this for a reason. (It was copied & pasted from Simorgh).
The 'success' conditions are different, that one just checks `params.BRANCH` (not `GIT_BRANCH`).

It's something to do with not spamming the Slack channel, but I don't know exactly what it does. 🤔 Worth documenting once we figure it out!

Reply:
`params.BRANCH` is chosen in the Jenkins Params interface. 

We have that defined in the Simorgh Infra Jenkins pipeline (https://github.com/BBC/simorgh-infrastructure/blob/latest/Jenkinsfile#L103-L104)  but we don't have this defined in this Psammead pipeline, so it can be removed.